examples/sn-dms-demo/src/components/UsersAndGroups/User/MembersList.tsx

```diff
@@ -0,0 +1,203 @@
+import Button from '@material-ui/core/Button'
+import Checkbox from '@material-ui/core/Checkbox'
+import TableCell from '@material-ui/core/TableCell'
+import { Content } from '@sensenet/client-core'
+import { ActionModel, GenericContent, Group, SchemaStore } from '@sensenet/default-content-types'
+import { Icon } from '@sensenet/icons-react'
+import { ContentList } from '@sensenet/list-controls-react'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { RouteComponentProps, withRouter } from 'react-router'
+import * as DMSActions from '../../../Actions'
+import { closeActionMenu, openActionMenu } from '../../../Actions'
+import { icons } from '../../../assets/icons'
+import { resources } from '../../../assets/resources'
+import { customSchema } from '../../../assets/schema'
+import { rootStateType } from '../../../store/rootReducer'
+import { selectUser, setActive, updateChildrenOptions } from '../../../store/usersandgroups/actions'
+import { DisplayNameCell } from '../../ContentList/CellTemplates/DisplayNameCell'
+import { DisplayNameMobileCell } from '../../ContentList/CellTemplates/DisplayNameMobileCell'
+import RemoveUserFromGroupDialog from '../../Dialogs/RemoveUserFromGroupDialog'
+
+const styles = {
+  deleteButton: {
+    fontFamily: 'Raleway Medium',
+    fontSize: 15,
+  },
+}
+
+interface MembersListProps extends RouteComponentProps<any> {
+  matchesDesktop: boolean
+}
+
+const mapStateToProps = (state: rootStateType) => {
+  return {
+    currentGroup: state.dms.usersAndGroups.group.currentGroup || ({} as Group),
+    childrenOptions: state.dms.usersAndGroups.user.grouplistOptions,
+    hostName: state.sensenet.session.repository ? state.sensenet.session.repository.repositoryUrl : '',
+    selected: state.dms.usersAndGroups.user.selected,
+    parent: state.dms.usersAndGroups.group.parent,
+    active: state.dms.usersAndGroups.user.active,
+    members: state.dms.usersAndGroups.group.members,
+  }
+}
+
+const mapDispatchToProps = {
+  openActionMenu,
+  closeActionMenu,
+  selectUser,
+  updateChildrenOptions,
+  openDialog: DMSActions.openDialog,
+  closeDialog: DMSActions.closeDialog,
+  setActive,
+}
+
+class MembersList extends Component<
+  ReturnType<typeof mapStateToProps> & typeof mapDispatchToProps & MembersListProps,
+  {}
+> {
+  constructor(props: MembersList['props']) {
+    super(props)
+
+    // this.handleRowDoubleClick = this.handleRowDoubleClick.bind(this)
```

Comment:
It might make sense to remove comments  and the constructor as well from this file.